tests/e2e/normalize.go

```diff
@@ -438,6 +438,9 @@ func normalizeHTTPResponses(t *testing.T, events test.LogEntries) {
 	visitor.replacePaths[".fingerprint"] = "abcdef0123A="
 	visitor.replacePaths[".startTime"] = "2024-04-01T12:34:56.123456Z"
 
+	// ServiceIdentity operations
+	visitor.replacePaths[".response.uniqueId"] = "12345678"
```

Comment:
Looks like we have two ways to override the uniqueID. I'll vote for the replace-by-kind approach
For Mock test development, one problem (the most time consuming part) is to figure out the polluted overridden fields.  
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/a9c4afddb4f51a9dac376335303ba34f17e5e5df/mockgcp/mockserviceusage/serviceusagev1beta1.go#L53